### ISSUE TITLE
about loss function

### ISSUE THREAD

**Issue by LeeCASC** [seanXYZ/SegDecNet#7]
In this paper, the author said the loss function is perfectly crossentropy, but when I modify the MSELoss to CrossEntropy Loss. The code do not work. How to modify the code better? Thank you!

**Comment by elepp** [comment]
use `criterion_segment = torch.nn.BCEWithLogitsLoss()` and take away `nn.ReLU(inplace=True)` in models.py at `self.layer5 = nn.Sequential(
            nn.Conv2d(1024, 1, 1),
            nn.ReLU(inplace=True)
        )`
